nodejs/awsx/ecs/container.ts

```diff
@@ -159,31 +159,349 @@ export interface KeyValuePair {
+     * a memory hard limit of 300 MiB. This configuration would allow the container to only reserve
+     * 128 MiB of memory from the remaining resources on the container instance, but also allow the
+     * container to consume more memory resources when needed.
+     *
+     * The Docker daemon reserves a minimum of 4 MiB of memory for a container, so you should not
+     * specify fewer than 4 MiB of memory for your containers.
+     *
+     * This parameter maps to `MemoryReservation` in the
+     * [Create-a-container](https://docs.docker.com/engine/api/v1.35/#operation/ContainerCreate)
+     * section of the [Docker-Remote-API](https://docs.docker.com/engine/api/v1.35/) and the
+     * `--memory-reservation` parameter to [docker-run](https://docs.docker.com/engine/reference/run/).
+     */
     memoryReservation?: pulumi.Input<number>;
+
+    /**
+     * The mount points for data volumes in your container.
+     *
+     * This parameter maps to `Volumes` in the
+     * [Create-a-container](https://docs.docker.com/engine/api/v1.35/#operation/ContainerCreate)
+     * section of the [Docker-Remote-API](https://docs.docker.com/engine/api/v1.35/) and the
+     * `--volume` parameter to [docker-run](https://docs.docker.com/engine/reference/run/).
+     */
     mountPoints?: pulumi.Input<aws.ecs.MountPoint[]>;
+
+    /**
+     * When this parameter is true, the container is given elevated privileges on the host container
+     * instance (similar to the root user).
+     *
+     * This parameter maps to `Privileged` in the
+     * [Create-a-container](https://docs.docker.com/engine/api/v1.35/#operation/ContainerCreate)
+     * section of the [Docker-Remote-API](https://docs.docker.com/engine/api/v1.35/) and the
+     * `--privileged` parameter to [docker-run](https://docs.docker.com/engine/reference/run/).
+     *
+     * Note: This parameter is not supported for Windows containers or tasks using the Fargate
+     * launch type.
+     */
     privileged?: pulumi.Input<boolean>;
+
+    /**
+     * When this parameter is true, the container is given read-only access to its root file system.
+     *
+     * This parameter maps to `ReadonlyRootfs` in the
+     * [Create-a-container](https://docs.docker.com/engine/api/v1.35/#operation/ContainerCreate)
+     * section of the [Docker-Remote-API](https://docs.docker.com/engine/api/v1.35/) and the
+     * `--read-only` parameter to [docker-run](https://docs.docker.com/engine/reference/run/).
+     *
+     * Note: This parameter is not supported for Windows containers.
+     */
     readonlyRootFilesystem?: pulumi.Input<boolean>;
+
+    /**
+     * The secrets to pass to the container. For more information, see
+     * [Specifying-Sensitive-Data](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data.html)
+     * in the Amazon Elastic Container Service Developer Guide.
+     */
+    secrets?: pulumi.Input<aws.ecs.Secret[]>;
```

Comment:
Only type-system change.  There is no runtime change since all we do is copy *all* these properties over.